src/hevm/src/EVM/UnitTest.hs

```diff
@@ -427,28 +427,31 @@ runUnitTestContract
                      (fromIntegral gasSpent :: Integer)
                in
                  pure
-                   ( "PASS " <> testName <> " (gas: " <> gasText <> ")"
+                   ("\x1b[32m[PASS]\x1b[0m "
+                   <> testName <> " (gas: " <> gasText <> ")"
                    , Right (passOutput vm dapp opts testName)
                    )
              (Right False, vm) ->
-               pure ("FAIL " <> testName, Left (failOutput vm dapp opts testName))
+               pure ("\x1b[31m[FAIL]\x1b[0m "
+               <> testName, Left (failOutput vm dapp opts testName))
              (Left _, _)       ->
-               pure ("OOPS " <> testName, Left ("VM error for " <> testName))
+               pure ("\x1b[33m[OOPS]\x1b[0m "
+               <> testName, Left ("VM error for " <> testName))
 
       let inform = \(x, y) -> Text.putStrLn x >> pure y
 
       -- Run all the test cases and print their status updates
       details <-
         mapM (\x -> runOne x >>= inform) testNames
 
-      let mails = [x | Right x <- details]
-      let fails = [x | Left x <- details]
+      let running = [x | Right x <- details]
+      let bailing = [x | Left x <- details]
```

Comment:
Surprised by this change.

Reply:
`fails` was actually misleading. The `running` tests contain both passing and failing tests, where the `bailing` tests are those with a vm error.